### PR TITLE
Add loading spinner SVG

### PR DESCRIPTION
**Pull request by kgcreative**
### Description
- add loading spinner svg and remove horizontal bar loading indicator
 
### Issues Resolved
solves line item: OpenSearch mark loading spinner in #372 
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by mihirsoni** [comment]
@kgcreative Could you sign your commits ?

File: src/core/server/rendering/views/template.tsx [hunk 1]
```diff
@@ -120,7 +148,7 @@ export const Template: FunctionComponent<Props> = ({
           data-test-subj="osdLoadingMessage"
         >
           <div className="osdLoaderWrap">
-            {openSearchLogo}
+            {openSearchLogoSpinner}
```
Comment (mihirsoni) [inline src/core/server/rendering/views/template.tsx:151]: Let's remove `opensearchLogo` if not needed.

**Comment by kgcreative** [comment]
> @kgcreative Could you sign your commits ?

Yep - I might close this PR and open a new one because I somehow messed up the rebase on local (yay git)

File: src/core/server/rendering/views/template.tsx (hunk 1 above)
Comment (kgcreative) [reply src/core/server/rendering/views/template.tsx:151]: It's still needed - it's referenced further down on a second use case where it's not a loading spinner but rather a static image with an error message.

**Comment by kgcreative** [comment]
Closing PR -- will open new one, properly signed.

**Pull request closed without merging**